public/index.html

```diff
@@ -2,11 +2,11 @@
 <html lang="en">
```

Comment:
maybe an idea to put all the `favicon` images in a seperate `favicon` folder in `public`

Reply:
a suggestion :p

Reply:
removed favicon pngs https://www.javatpoint.com/html-favicon